runtime/near-vm/compiler/src/function.rs

```diff
@@ -4,6 +4,8 @@
 //! A `Compilation` contains the compiled function bodies for a WebAssembly
 //! module (`CompiledFunction`).
 
+// cspell:ignore rkyv, rodata
```

Comment:
Thought: crate names like `rkyv` seem like they should probably go into the global list unconditionally.